lib/cli_miami.rb

```diff
@@ -7,6 +7,6 @@
 # the core initilization runs from `namespaced.rb`
 require 'namespaced'
```

Comment:
this requires the `namespaced.rb` file in the `lib` dir.  Its some DSL magic :) By default, it creates a class just called `A` and another class called `S`.  The `A` class has a class methods call `.sk` and the `S` class has a method called `.ay`.  This makes the API cleverly `A.sk` and `S.ay`.

however if you want to namespace it to protect your code (or personal preference), in your Gemfile for example, you can either do

```ruby
# this creates the `A` and `S` classes
gem 'cli-miami'

# this creates namespaced CliMiami::A and CliMiami::S classes
gem 'cli-miami', require: 'namespaced'
```